Change to F4PGA

xanjohns opened the pull request:
Signed-off-by: Xan Johnson <xanjohns@gmail.com>

<!--- A standard pull request template to use for all Symbiflow repositories. Adapted from Open Source Templates(https://www.talater.com/open-source-templates/#/) by Tal Ater. -->

<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->

Replaced references and links involving SymbiFlow to their appropriate F4PGA counterparts. Also replaced syncing target repositories to new targets in the [Chips Alliance](https://github.com/chipsalliance) organization.

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

xanjohns commented:
What would the new email be in this case? Is it just f4pga-wg@lists.chipsalliance.org?

https://github.com/SymbiFlow/symbiflow-common-config/blob/6b18deba76d2ffd0cdb0887c631efb49b6b927a2/CODE_OF_CONDUCT.md?plain=1#L48-L49

umarcor commented on .github/sync.yml:
```diff
@@ -1,13 +1,10 @@
-      - SymbiFlow/symbiflow-arch-defs
-      - SymbiFlow/symbiflow-examples
-      - SymbiFlow/yosys-symbiflow-plugins
+      - chipsalliance/fpga-interchange-schema
```
Can you please provide some context about the motivation to update some refs but remove others?

xanjohns replied:
In our sync meeting two weeks ago, we discussed only targeting repositories that have been fully moved over to chipsalliance and converted to the new license. These were the repositories proposed by @acomodi

umarcor replied:
Thanks for explaining! You can add chipsalliance/f4pga-v2x as well.

umarcor commented on CONTRIBUTING.md:
```diff
@@ -11,8 +11,8 @@ All contributions should be sent as
-Symbiflow repositories are licensed under the very permissive
-[ISC Licence](https://opensource.org/licenses/ISC). A copy can be found in the [`LICENSE`](LICENSE) file.
+F4PGA repositories are licensed under the very permissive
+[Apache 2.0 Licence](https://opensource.org/licenses/Apache-2.0). A copy can be found in the [`LICENSE`](LICENSE) file.
```
The current LICENSE file corresponds to ISC. Please update it in this PR.

xanjohns replied:
The repositories that this file syncs with, listed in `.github/sync.yml`, all have the Apache 2.0 License, which is why I have updated it here, since this is the file that will be copied over to those target repositories. I understand the confusion though, since the `symbiflow-common-config` repository does not have this license. Maybe the best solution would be to make another `CONTRIBUTING.md` file for this repository, then rename the `CONTRIBUTING.md` file that will be used to sync to other repositories. Let me know what you think. @umarcor

umarcor replied:
Actually, I meant to change https://github.com/SymbiFlow/symbiflow-common-config/blob/main/LICENSE, because the license of `symbiflow-common-config` does also need to be Apache 2.0.

I see the misunderstanding you are explaining. I do think it would be sensible to make it explicit which are the "assets" and which are the sources corresponding to the `symbiflow-common-config` repository itself. Therefore, I would suggest to go ahead and rename the current `CONTRIBUTING.md` to something else (say `CONTRIBUTING.ref.md`) or move it into the assets subdir. However, I don't know if the Action being used ATM does support the files having a different name or being in a different location. A different (or custom, https://github.com/SymbiFlow/actions) Action might be required in order to support such use case.

umarcor commented:
> What would the new email be in this case? Is it just [f4pga-wg@lists.chipsalliance.org](mailto:f4pga-wg@lists.chipsalliance.org)?

I think so!

xanjohns commented:
Our current Github Action does support renaming files upon syncing, so I've gone ahead and renamed `CONTRIBUTING.md` to `CONTRIBUTING.ref.md`. I've also updated the email and added the other repository that you suggested we sync to. Should now be good to go. @umarcor

umarcor reviewed (approved):
LGTM!
